lib/sqlite/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.5)
+PROJECT(sqlite)
+
+SET(SOURCES sqlite3.c)
+
+ADD_DEFINITIONS(-D_GNU_SOURCE)
+
+ADD_LIBRARY(sqlite STATIC ${SOURCES})

```

Comment:
You can try about the debian build error on Travis to add:
```
set_property(TARGET sqlite PROPERTY POSITION_INDEPENDENT_CODE ON)
```
Possible that it can fail without also on other Linux distributions.

Reply:
Thank you very much for the help. Build is green now.